src/mscorlib/shared/System/Reflection/ReflectionTypeLoadException.cs

```diff
@@ -51,9 +51,15 @@ public override string Message
 
                 StringBuilder text = new StringBuilder();
                 text.AppendLine(base.Message);
-                foreach (Exception e in LoaderExceptions)
+                if (LoaderExceptions != null)
```

Comment:
The null check needs to happen before `LoaderException.Length == 0` above.

Reply:
Now that the null check has been added above, this one can be removed.